src/Components/ModelledPropertyBuilder/ModelledPropertyBuilder.tsx

```diff
@@ -261,16 +261,18 @@ const ModelledPropertyBuilder: React.FC<ModelledPropertyBuilderProps> = ({
                             }}
                         />
                     )}
+                </Stack>
+                <Stack horizontal>
                     {isClearEnabled && (
                         <ActionButton
```

Comment:
can we give it a classname as well in case someone wants to move it around specifically? you can do that by simply adding a property in the `getStyles` funciton and having it only include the `[classNamePrefix.whatever]` and just have no styles there.

Reply:
Like 
```ts
clearButton: [classNames.clearButton],
```